pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -11,11 +11,32 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 )
 
+// OOBInvitation to connect with did-exchange.
+type OOBInvitation struct {
```

Comment:
naming suggestions are welcome

Reply:
Looks like this is going to be renamed to `Invitation` as per [this](https://github.com/hyperledger/aries-framework-go/pull/1520/files#diff-81d07647d9d693299e624e7e5d843199R38) TODO.

Reply:
Something like that, yes.